network.loki.Session.metainfo.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>network.loki.Session.desktop</id>
+  <name>Session Desktop</name>
+  <project_license>GPL-3.0</project_license>
+  <developer_name>Loki Project</developer_name>
+  <summary>Private messenger</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://getsession.org/</url>
+  <description>
+    <p>
+      Session is a new breed of private messenger, built on a unique network of
+      user-operated servers spread all over the world. With no central servers, 
+      Session can’t leak or sell your data. And with Session’s private routing 
+      protocols, your messages are completely anonymous. No one ever knows who 
+      you’re talking to, what you’re saying, or even your IP address.
+    </p>
+    <ul>
+      <li>
+        Fully anonymous account creation: No phone number or email needed to create a Session ID
+      </li>
+      <li>
+        Decentralised server network: No data breaches, no central point of failure
+      </li>
+      <li>
+        No metadata logging: Session doesn't store, track, or log your messaging metadata
+      </li>
+      <li>
+        IP address protection: Device IP addresses are never exposed to the person you're talking
+        to or the servers holding your data
+      </li>
+      <li>
+        Closed groups: Private, end-to-end encrypted group chats for up to 10 people
+      </li>
+      <li>
+        Secure attachments: Share voice snippets, photos, and files with Session's secure encryption
+        and privacy protections
+      </li>
+      <li>
+        Multi-device syncing: One Session ID for your phone and computer
+      </li>
+      <li>
+        Free and open-source: Don’t take our word for it — check Session's code yourself
+      </li>
+    </ul>
+    <p>
+      Session is free as in free speech, free as in free beer, and free of ads and
+      trackers. Session is built and maintained by the Loki Foundation, Australia’s
+      first privacy tech not-for-profit organisation. Take back your online privacy
+      today — download Session.
+    </p>
+    <p>
+      Want to build from source, report a bug, or just take a look at our code? Check
+      out Session on GitHub: https://github.com/loki-project/session-desktop
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://i.imgur.com/ZnHvYjo.jpg</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="1.0.8" date="2020-06-01"/>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
```

Comment:
You can remove the ones with none as a value, it's the default one

Reply:
Should be resolved as of 0e40e124f2332f0b245358615702301a2b019cb0